### PR TITLE
Adds slices for banana cream pie and mime tart

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
Makes both pies sliceable and adds sprites

~~Note: there are in-hand sprites for the mime tart as well. But that would require moving to a new folder. This seemed unnecessary but if it is desirable, let me know and I will make the adjustment. (currently defaults to the yellow basic pie sprite)~~

## Why / Balance
They were missing.

## Technical details
n/a

## Media
image

- [x] I have added screenshots/videos to this PR showcasing its changes in-game, **or** this PR does not require an in-game showcase

## Breaking changes
n/a

### CODE REVIEW

**Comment by deltanedas** [comment]
> But that would require moving to a new folder.

item has heldPrefix for this

**Comment by CMDR-Piboy314** [comment]
do slices of banana pie still splat?

**Comment by IamVelcroboy** [comment]
> do slices of banana pie still splat?

they do not. You still need a whole pie for that

**Comment by IamVelcroboy** [comment]
I've ignored the test fail because I don't think it's relevant. Let me know if it is and I just missed something.

**Comment by Arteben** [comment]
I tested it and it is normal, simple it adds some slices of pies.

**Comment by Arteben** [comment]
Need add description in change log
image
example Added slices of mime and banana pies or something else

**Comment by Partmedia** [comment]
I know this is your editor doing this, but reindenting the whole file and/or trimming spaces makes it harder to review what you actually changed. Please be aware of this and the [pull request guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html#content) next time.

**Comment by IamVelcroboy** [comment]
> Need add description in change log example Added slices of mime and banana pies or something else

OK done

**Comment by IamVelcroboy** [comment]
> I know this is your editor doing this, but reindenting the whole file and/or trimming spaces makes it harder to review what you actually changed. Please be aware of this and the [pull request guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html#content) next time.

Yeah, that's annoying. Sorry about that.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml [hunk 1]
```diff
@@ -348,6 +364,37 @@
...
+  - type: Sprite
+    state: mime
+  - type: Item
+    heldPrefix: mime
+  - type: SliceableFood
+    slice: FoodTartMimeSlice
+
+- type: entity
+  name: slice of mime tart
+  parent: FoodPieSliceBase
```
Comment (Partmedia) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pie.yml:385]: Missing description, but it's fine

**Review by Partmedia — APPROVED** [review]
Thank you for your contribution. In the future, please remember to review your diff to make sure you have not made any unintentional changes. Please also review the [Changelog guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html#writing-an-effective-changelog).

**Pull request merged**